app/routers/event.py

```diff
@@ -1,12 +1,24 @@
+import re
+from datetime import datetime as dt
 from operator import attrgetter
 from typing import List
 
 from fastapi import APIRouter, Request
+from fastapi import Depends
+from fastapi.responses import RedirectResponse
+from starlette.status import HTTP_303_SEE_OTHER
 
+from app.database.database import get_db
 from app.database.models import Event
+from app.database.models import User
 from app.database.models import UserEvent
 from app.dependencies import templates
+from app.internal.event import validate_zoom_link
 from app.internal.utils import create_model
+from app.routers.user import create_user
+
+VALID_MAIL_REGEX = re.compile(r'^\S+@\S+\.\S+$')
```

Comment:
You can use EmailStr of Pydantic (I think we even have a validation method in another file on this project)